tests/_utils.rs

```diff
@@ -74,16 +74,16 @@ impl Default for SampleWorkspace {
     fn default() -> Self {
         let temp = Temp::new_dir().expect("create working directory for sample workspace");
 
-        MergeableFileSystemTree::<&str, &str>::from(dir! {
+        MergeableFileSystemTree::<&str, String>::from(dir! {
             "flat" => dir! {
                 "0" => file!("")
-                "1" => file!("a")
-                "2" => file!("ab")
-                "3" => file!("abc")
+                "1" => file!("a".repeat(1000))
+                "2" => file!("a".repeat(2000))
+                "3" => file!("a".repeat(3000))
             }
             "nested" => dir! {
                 "0" => dir! {
-                    "1" => file!("abcdef")
+                    "1" => file!("a".repeat(6000))
```

Comment:
@peret 

> Wait... the point of my original change in `tests/_utils.rs` was to make flat/ and nested/ have different sizes, so the original fault in the test case gets triggered. Now you changed it back so both folders are the same size again. It now gets reported as about 6k bytes instead of 6 bytes, but they're the same size again.

Can you try apply this change locally to see if it still passes?

```suggestion
                    "1" => file!("a".repeat(5000))
```

Reply:
@KSXGitHub 
Yes, that works.

Reply:
I think the reason why it still passes with 6000 was because directories have apparent size greater than zero. And `nested/0` is 2 directories compared to 1 directory of `flat`. So if you run the test with output printed out, so should see `nested` being place below `flat`. Can you test my hypothesis? I'm curious.